### PR TITLE
docs(managers): add missing url and displayName

### PR DESCRIPTION
**Pull request by risu729**
## Changes

Updates the docs of managers. Mainly adds missing URLs of the tools.

Also, it adds `displayName` if the default (title case split by `-`) isn't appropriate.

My apologies for reordering some codes. Please let me know if it's better to preserve the order.

This PR updates the codes in `lib/modules/managers/` but all changes should affect only docs.

## Documentation (please check one with an [x])

- [x] I have updated the documentation, or
- [ ] No documentation update is required

## How I've tested my work (please select one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

File: lib/modules/manager/bitbucket-pipelines/index.ts [hunk 1]
```diff
@@ -4,14 +4,15 @@ import { extractPackageFile } from './extract';
 
 export { extractPackageFile };
 
+export const url =
+  'https://support.atlassian.com/bitbucket-cloud/docs/get-started-with-bitbucket-pipelines/';
+export const categories: Category[] = ['ci'];
+export const urls = [
```
Comment (viceice) [inline lib/modules/manager/bitbucket-pipelines/index.ts:10]: do we support url and urls? 🤔

File: lib/modules/manager/maven/index.ts [hunk 2]
```diff
@@ -5,6 +5,9 @@ import * as mavenVersioning from '../../versioning/maven';
 export { extractAllPackageFiles } from './extract';
 export { bumpPackageVersion, updateDependency } from './update';
 
+export const url = 'https://maven.apache.org/index.html';
```
Comment (viceice) [inline lib/modules/manager/maven/index.ts:8]:
```suggestion
export const url = 'https://maven.apache.org';
```
I would drop the index file

File: lib/modules/manager/meteor/index.ts [hunk 3]
```diff
@@ -3,10 +3,11 @@ import { NpmDatasource } from '../../datasource/npm';
 
 export { extractPackageFile } from './extract';
 
+export const url = 'https://docs.meteor.com/';
```
Comment (viceice) [inline lib/modules/manager/meteor/index.ts:6]: maybe we should also drop trailing slash

File: lib/modules/manager/bitbucket-pipelines/index.ts (hunk 1 above)
Comment (risu729) [reply lib/modules/manager/bitbucket-pipelines/index.ts:10]: `url` is for the name, and `urls` is for the references.

`url`
image
`urls`
image

**Review by viceice — APPROVED** [review]

**Pull request merged**